src/loki/search.cc

```diff
@@ -380,30 +374,18 @@ struct ProjectPoint {
     const auto sq_distance = approx.DistanceSquared(point);
     if(sq_distance < sq_closest_distance) {
       // this block is not in the hot spot
-      b->closest_segment = segment;
+      closest_segment = segment;
       sq_closest_distance = sq_distance;
-      b->closest_point = point;
+      closest_point = point;
     }
   }
 
- private:
-  // To limit the size of the struct and optimize cache locality of
-  // the data used in the hot spot, we box the data not used in the
-  // hot spot.
-  struct Box {
```

Comment:
in testing using `valhalla_benchmark_loki` the removal of this object actually sped up the code. for searches of a single location there was no change in speed. for searches where there more locations (like 200 or so) the code was actually faster with this removed